package.json

```diff
@@ -7,7 +7,8 @@
     "build": "webpack -d --watch",
     "start": "nodemon server/server.js",
     "seedDB": "node fakeDataGenerator.js && mysql -u root < schema.sql",
-    "test": "jest"
+    "test": "jest",
+    "test:coverage": "npx jest --coverage"
```

Comment:
oh whoa what's the difference between 'test' and 'test:coverage'?

Reply:
It gives a "code coverage" report after the test runs letting you know how much of your code is being "tested".